app/components/Edit/EditSidebar.jsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withRouter } from 'react-router';
+
+class EditSidebar extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    const resource = this.props.resource;
+    let actionButtons = [
+      <button
+        className="edit--aside--content--button"
+        key="submit"
+        disabled={this.props.submitting}
+        onClick={this.props.handleSubmit}
+      >Save Changes</button>,
+      <button
+        className="edit--aside--content--button cancel--button"
+        key="cancel"
+        onClick={this.props.handleCancel}
+      >Discard Changes</button>,
+      <button
+        className="edit--aside--content--button deactivate--button"
+        key="deactive"
+        disabled={this.props.submitting}
+        onClick={() => this.props.handleDeactivation('resource', resource.id)}
+      >Deactivate</button>,
+    ];
+    if (this.props.newResource) {
+      actionButtons = [
+        <button
+          className="edit--aside--content--button"
+          key="submit"
+          disabled={this.props.submitting}
+          onClick={this.props.createResource}
+        >Submit</button>,
+        <button
+          className="edit--aside--content--button cancel--button"
+          key="cancel"
+          onClick={this.props.handleCancel}
+        >Cancel</button>,
+      ];
+    }
+    if (!resource.certified) {
+      actionButtons.push(
+        <button
+          className="edit--aside--content--button hap--button"
+          key="hap"
+          onClick={this.props.certifyHAP}
+        >HAP Approve</button>,
+      );
+    }
+    return (
+      <nav className="sidebar">
+        <ul className="sidebar--org--info">
+          <li className="sidebar--list--heading"><h3>Organization</h3></li>
+          <li className="sidebar--organization--name">{resource.name}</li>
+        </ul>
+        <ul>
+          <hr />
+          <li className="sidebar--list--heading"><h3><a href="#services">Services</a></h3></li>
+          {resource.services.map(service => (
```

Comment:
For useability, these could be anchor tags that link to the top of that individual service section on the page.

I notice this doesn't update as I add a new service, which I guess would also be useful (serviceNameField.onblur updates the list state?)

Reply:
Thanks for the feedback. Will do.